src/cf-expandables/src/cf-expandables.js

```diff
@@ -1,117 +1,8 @@
-/**
- * cf-expandables
- * https://github.com/cfpb/cf-expandables
- *
- * A public domain work of the Consumer Financial Protection Bureau
- */
+'use strict';
 
-global.jQuery = require('jquery');
-require('jquery.easing');
+var Expandable = require( './Expandable' );
 
-(function( $ ) {
+// polyfill for ie9 compatibility
+require( 'classlist-polyfill' );
```

Comment:
Do you mean `AtomicComponent/src/utilities/dom-class-list/`? Maybe the way I'm `require`-ing it is preventing it from working properly? I was having problems in IE9 that fixed themselves with the polyfill, but I'm all ears on an easier way to fix them. 😄